syntax_checkers/go/govet.vim

```diff
@@ -23,7 +23,7 @@ function! SyntaxCheckers_go_govet_IsAvailable() dict
 endfunction
 
 function! SyntaxCheckers_go_govet_GetLocList() dict
-    let makeprg = self.getExecEscaped() . ' vet'
+    let makeprg = self.makeprgBuild({ 'args_after': 'vet' })
```

Comment:
and now I read https://github.com/scrooloose/syntastic/wiki/Syntax-Checker-Guide , looks like `args` can be changed by user and `args_before` and `args_after` can't. So I have decided to use `args_before`.